mods/empyrean_campaign/cutscenes/ending.txt

```diff
@@ -6,7 +6,7 @@ music=music/unrest_theme.ogg
 image=images/menus/backgrounds/badlands.png,1
 caption=At long last, your journey across the surface and into the Underworld has come to an end.
 pause=8s
-caption=Upon entering The Pit, a glowing obelisk shined in the distance.
+caption=Upon entering The Pit, a glowing obelisk shone in the distance.
```

Comment:
This is not an error. Both "shined" and "shone" are valid past-tense forms of shine.

Reply:
There is a difference in meaning though. "shone" is what the sun did, "shined" is what the shoe-shine boy did. The obelisk is emitting a shine itself, not making something else shine.

At least that's how I am familiar with the word - maybe this distinction gut blurred during recent years.